app/lib/alimonitor-services/helpers/ServicesDataCommons.js

```diff
@@ -16,6 +16,22 @@
 const Utils = require('../../utils');
 const { rctData } = require('../../config/configProvider.js');
 
+/**
+ * PERIOD_NAME_REGEX is regualar expersion for examining
```

Comment:
`PERIOD_NAME_REGEX` is a regular expression
(missing `a` and a typo in `regular`)
...
can be considered period name
(without `as`)

Reply:
done